onnx/defs/rnn/defs.cc

```diff
@@ -56,16 +59,31 @@ via some custom implementation such as CuDNN.
 `RBi` - R recurrence weight matrix for backward input gate
 `WBbi` - WR bias vectors for backward input gate
 `RBbi` - RR bias vectors for backward input gate
-`ReLU(X)` - max(X, 0)
-`tanh(X)` - hyperbolic tangent of X
 `H` - Hidden state
 `num_directions` - 2 if direction == bidirectional else 1
 
+Activation functions:
```

Comment:
put them in the common block?

Reply:
I tried it using replaceAll(). It looked quite awkward. I will keep it as is, unless you have a better way of doing it.